src/app/modules/group/pages/group-details/group-details.component.html

```diff
@@ -10,13 +10,13 @@
     ></alg-pending-join-requests>
     <alg-group-indicator
         class="group-indicator"
-        *ngIf="group?.currentUserManagership !== 'none' || group?.currentUserMembership !== 'none'"
+        *ngIf="group.currentUserManagership !== 'none' || group.currentUserMembership !== 'none'"
         [group]="group"
     >
     </alg-group-indicator>
     <!-- tabs -- if only the first tab is visible, do not show the tab bar -->
-    <div class="nav-tab" *ngIf="group.isCurrentUserManager">
-      <div
+    <div class="nav-tab" *ngIf="group.isCurrentUserManager || group.currentUserManagership !== 'none' || group.currentUserMembership !== 'none'">
```

Comment:
`group.isCurrentUserManager` covers `group.currentUserManagership !== 'none'` so it is useless here. 
And as usual, you also need to display the tab if the user arrives on the page via a link... in this case, you have to show the tab (as long as he has not clicked on another tab) and display in the content that he cannot access this content. -> you did well already recently for other tabs a few days ago

Reply:
Fixed